OpenOversight/tests/test_utils.py

```diff
@@ -53,13 +53,7 @@ def test_filter_by_badge_no(mockdata):
         assert '12' in str(assignment.star_no)
 
 
-def test_allowed_filenames(app):
-    extension = ['png', 'jpg', 'jpeg', 'mpeg', 'mp4']
-    for e in extension:
-        test_filename = 'test.{}'.format(e)
-	assert OpenOversight.app.utils.allowed_file(test_filename) == True
-
-
-def test_not_allowed_filenames(app):
-    test_filename = 'test.gif'
-    assert OpenOversight.app.utils.allowed_file(test_filename) == False
+def test_hash_file(mockdata):
+    hash_result = OpenOversight.app.utils.hash_file('bacon')
```

Comment:
where is the file we are expecting to hash `bacon` actually coming from? the test data directory?